src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,512 @@
+
+Further PET datasets are available from [OpenNeuro](https://openneuro.org).
+
+## PET recording data
+
+As specified above, this extension is relevant for brain PET imaging
+and its associated data, such as blood data.
+In addition, the extension is in accordance with the guidelines for reporting
+and sharing brain PET data (Knudsen et al. 2020, [1]).
+If you want to share structural magnetic resonance (MR) images with your PET data,
+please distribute them according to the original BIDS specification
+(https://bids-specification.readthedocs.io/en/stable).
+Please pay specific attention to the format the MR images are in,
+such as if they have been unwarped in order to correct for gradient non-linearities.
+There is a specific field in the MRI BIDS (https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/01-magnetic-resonance-imaging-data.html) called “NonlinearGradientCorrection”
+which indicates this.
+The reason for this is that the MRI needs to be corrected for nonlinear gradients
+in order to fit the accompanying PET scans for co-registration [1, 2].
+In general, SI units must be used (we refer to https://bids-specification.readthedocs.io/en/stable/99-appendices/05-units.html) and we recommend to use the CMIXF style formatting for SI units, for example, "kBq/mL"
+rather than "kilobecquerel per ml".
+An overview of a common PET experiment (with blood data) can be seen in Figure 1,
+defined on a single time scale relative to a predefined “time zero”
+(which should be defined either to be scan start or injection time;
+please note an exception to this definition is possible if a pharmacological
+within-scan challenge is performed).
+
+![Figure 1](images/PET_scan_overview.png "Overview of PET recording data")
+
+Figure 1: Overview of a common PET experiment, including blood measurements,
+and defined on a common time scale.
+Note, “time zero” can either be defined as time of injection or scan start,
+and all the PET and blood data should be decay-corrected to this time point.
+Furthermore, although in this example tracer injection coincides with scan start,
+this is not always the case and hence we allow for the flexibility of specifying
+either time of injection or scan start as “time zero”.
+
+Template:
+
+```Text
+sub-<participant_label>/
+      [ses-<session_label>/]
+        pet/
+        sub-<participant_label>[_ses-<session_label>][_task-<task_label>][_acq-<label>][_rec-<label>][_run-<index>]_pet.nii[.gz]
+        sub-<participant_label>[_ses-<session_label>][_task-<task_label>][_acq-<label>][_rec-<label>][_run-<index>]_pet.json
+```
+
+PET data belongs in the /pet folder. PET imaging data SHOULD be stored in 4D
+(or 3D if only one volume was acquired) NifTI files with `_pet` suffix.
+Volumes should be stored in chronological order (the order they were acquired in).
+
+Multiple sessions (visits) are encoded by adding an extra layer of directories
```

Comment:
Hi @Remi-Gau, yes, we will refer to the common principles section, but we also want to be very clear here for PET researchers what constitutes a session and a run, since this is somewhat different in PET than in MRI. Does that make sense. Especially in PET we can take a person out of the scanner during a run and then put them in again (e.g. during a 90 minute scan if they need to use the bathroom) and call it the same run, which you just wouldn't do for MRI. Cheers, Mel